okonomiyaki/file_formats/tests/test_legacy.py

```diff
@@ -0,0 +1,77 @@
+import sys
+
+if sys.version_info < (2, 7):  # noqa
+    import unittest2 as unittest
+else:
+    import unittest
+
+from okonomiyaki.platforms import EPDPlatform
+
+from ..legacy import _guess_platform_abi
+
+
+class TestGuessPlatformABI(unittest.TestCase):
+    def test_brood_cases(self):
+        # set of (python_tag, platform string) pairs that have at least one
+        # index in our current instance of packages.e.com
+        tag_platform = {
+            ('cp27', 'osx_x86'): 'darwin',
```

Comment:
The `python_tag, platform` pairs were computed w/ SQL from a dump of our prod database. The pair -> abi is manually curated, assuming we use the same abi as the python.org installers on windows.